cpp/src/utilities/cython.cpp

```diff
@@ -124,7 +179,8 @@ void populate_graph_container(graph_container_t& graph_container,
         graph_props,
         sorted_by_global_degree_within_vertex_partition,
         do_expensive_check);
-      graph_container.graph_ptr_union.graph_view_t_double_mg_ptr = g;
+      graph_container.graph_ptr_union.graph_view_t_double_mg_ptr =
+        std::unique_ptr<experimental::graph_view_t<int, int, double, false, true>>(g);
```

Comment:
Have you considered using std::make_unique?  This would clean u p this block of code a little bit (don't need the temporary variable g).

Reply:
done